test/kixi/hecuba/weather_test.clj

```diff
@@ -2,6 +2,4 @@
   (:require [clojure.test :refer :all]
             [kixi.hecuba.weather :refer :all]))
 
-(deftest a-test
-  (testing "FIXME, I fail."
-    (is (= 0 1))))
+
```

Comment:
Some tests (for things like the McKiver degree day function and some of the other pure functions) would be good.